grove/measurements/estimation.py

```diff
@@ -180,6 +181,7 @@ def estimate_pauli_sum(pauli_terms, basis_transform_dict, program,
         sample_variance = coeff_vec.T.dot(covariance_mat).dot(coeff_vec) / results.shape[1]
 
     return EstimationResult(expected_value=coeff_vec.T.dot(np.mean(results, axis=1)),
+                            term_wise_expected_value=np.mean(results, axis=1),
```

Comment:
From the name `term_wise_expected_value` it is not obvious why you would not want to multiply each mean by the coefficient (without summing at the end). How about we name it `pauli_moments` or `pauli_expectations` or something like that to indicate that they are the bare Paulis.

Reply:
:bear: :+1: